ietf/doc/storage_backends.py

```diff
@@ -22,7 +22,37 @@ class CustomS3Storage(S3Storage):
 
     def __init__(self, **settings):
         self.in_flight_custom_metadata: Dict[str, Dict[str, str]] = {}
-        return super().__init__(**settings)
+        super().__init__(**settings)
+
+    def get_default_settings(self):
+        # add a default for the ietf_log_blob_timing boolean
+        return super().get_default_settings() | {"ietf_log_blob_timing": False}
+
+    def _save(self, name, content):
+        # Only overriding this to add the option to time the operation
+        before = timezone.now()
+        result = super()._save(name, content)
```

Comment:
this can raise - do we want to catch and still log timing?

Reply:
Yes, and done